ReachRecord[] records
float32 reach_percentage
float32 total_pose_score
float32 norm_total_pose_score
float32 avg_num_neighbors
float32 avg_joint_distance
